build.sh

```diff
@@ -30,11 +30,20 @@ then
   for api in $apis
   do 
     [[ -d "$api" ]] && apidir=$api || apidir=apis/$api
-    dotnet sln AllProjects.sln add $apidir/*/*.csproj
-  
+    for project in $apidir/*/*.csproj
```

Comment:
Any reason we don't want to build the analyzers here? Aren't they as portable as we'd like?